doc/onvif-user-authentication.md

```diff
@@ -31,7 +31,7 @@ The AuthMode can be:
 * both
 * none
 
-SecretPath should contain:
+SecretName should contain:
 * username
 * password
 
```

Comment:
Lines 44 and 50 use the word path in the configuration.toml

Reply:
@vli11 , @ChristianDarr-personal is referring to the in-line TOML snippet below, not the actual configuration file.